src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/8.sql

```diff
@@ -0,0 +1,2441 @@
+
+/*************************************************************************************************
+    Auto-Generated from Sql build task. Do not manually edit it. 
```

Comment:
8.sql doesn't have the `Configure database` section

Reply:
it just got reformatted